dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/InternalLogger.kt

```diff
@@ -96,6 +96,13 @@ interface InternalLogger {
         additionalProperties: Map<String, Any?>? = null
     )
 
+    /**
+     * Logs a metric from the internal implementation.
```

Comment:
what is the meaning of `internal implementation` here? Also maybe we need to add the statement that metric will be sent to the telemetry (and say how), because from the method arguments it is not evident where is the metric here.